docs/node/mainnet/README.md

```diff
@@ -56,6 +56,27 @@ for the current Testnet parameters.
 
 This section contains parameters for various ParaTimes known to be deployed on the Mainnet.
 
+### Sapphire
```

Comment:
While you're at it, can you also swap the Emerald-Cipher menu entries in https://github.com/oasisprotocol/docs/blob/main/sidebarDapp.ts#L54-L83 ?